src/main/webapp/app/course/manage/course-update.component.html

```diff
@@ -47,6 +47,22 @@ <h4 *ngIf="course.id" id="jhi-course-heading-edit" jhiTranslate="artemisApp.cour
                         </div>
                     </ng-container>
                 </div>
+                <div class="form-group" *ngIf="course && course.maxComplaints > 0">
+                    <label class="form-control-label" jhiTranslate="artemisApp.course.maxComplaints" for="field_maxComplaints">Maximum Number of Complaints</label>
+                    <input required type="number" class="form-control" name="maxComplaints" id="field_maxComplaints" formControlName="maxComplaints" />
+                </div>
+                <div class="form-group">
+                    <label class="form-control-label" jhiTranslate="artemisApp.course.maxComplaintTimeDays" for="field_maxComplaintTimeDays"
+                        >Deadline for complaints in days after result date</label
+                    >
+                    <input required type="number" class="form-control" name="maxComplaintTimeDays" id="field_maxComplaintTimeDays" formControlName="maxComplaintTimeDays" />
+                </div>
+                <div class="form-check">
+                    <input type="checkbox" class="form-check-input" name="studentQuestionsEnabled" id="field_studentQuestionsEnabled" formControlName="studentQuestionsEnabled" />
+                    <label class="form-control-label" jhiTranslate="artemisApp.course.studentQuestionsEnabled" for="field_studentQuestionsEnabled"
```

Comment:
please add a switch button to enable / disable the whole feature.
If it is enabled, only positive values are allowed.
Disabled basically sets both values to 0 and then means that the complaint and more feedback request functionality is not displayed in the rest of the application